src/modules/configurator/components/ConfigDisplay.tsx

```diff
@@ -1,20 +1,35 @@
-import { currentConfigurationsState } from "../state";
 import { ConfigNavbar } from "./ConfigNavbar";
 import { ConfigEdit } from "./ConfigEdit";
+import { ConfigurationSummary } from "./ConfigurationSummary";
 import { useRecoilValue } from "recoil";
-import { configuratorRoutes } from "./const";
+import { activePageState, userConfigurationState } from "../state";
+import { PageLoading } from "../../global/components";
+import { useConfigEdit } from "../hooks";
 
 export function ConfigDisplay() {
-  const configurations = useRecoilValue(currentConfigurationsState);
+  const activePage = useRecoilValue(activePageState);
+  const configuration = useRecoilValue(userConfigurationState);
+
+  const { isDropdownOpen, activeDropdownName } = useConfigEdit();
+
+  if (!configuration) return <PageLoading />;
 
   return (
     <section className="relative h-full">
       <ConfigNavbar
-        returnPath={configuratorRoutes.carSelect}
-        model={configurations?.model}
-        productionYear={configurations?.productionYear}
+        model={configuration.model}
+        productionYear={configuration.productionYear}
+        returnPath=""
+        isConfigurationEdit
       />
-      <ConfigEdit />
+      {activePage && activePage.name === "Summary" ? (
```

Comment:
`activePage?.name === "Summary"` is shorter

Reply:
Replaced with activePage?.name.